pkg/schedule/prepare_checker.go

```diff
@@ -47,6 +49,11 @@ func (checker *prepareChecker) check(c *core.BasicCluster) bool {
 		checker.prepared = true
 		return true
 	}
+	if float64(c.GetStaleRegionCnt()) < float64(c.GetTotalRegionCount())*(1-collectFactor) {
+		log.Info("stale meta region number is satisfied, skip prepare checker", zap.Int64("stale-region", c.GetStaleRegionCnt()), zap.Int("total-region", c.GetTotalRegionCount()))
+		checker.prepared = true
+		return true
+	}
 	// The number of active regions should be more than total region of all stores * collectFactor
 	if float64(c.GetTotalRegionCount())*collectFactor > float64(checker.sum) {
```

Comment:
Do we still need it?

Reply:
maybe it is better to remove them in another pr? This pr just focuses on accelerating coordinator.